### PR TITLE
fix width

### LINKED ISSUE #8: BUG: Profits table does not fill its whole frame
### Have you updated the title?

- [X] I have updated the issue's title.

### Summary

After navigating to a commons, the Profits table contains whitespace instead of taking up the whole frame.

### Expected Behavior

We want the Profits table to not have any whitespace, and it should take up the whole frame.

### Current / Observed Behavior

Screenshot 2024-11-14 at 6 11 11 PM

There is extra whitespace on the Profits table

### Steps to Reproduce

1. Navigate to a Common
2. View the Profits column

### Browser / Platform Info

_No response_

### PR DESCRIPTION
**Pull request by blobthebuilder**
## Overview

In this PR, I fix the width of the Profit Table so that there is no empty white space. It is also still responsive to overflow.

## Screenshots (Optional)
image

## Validation (Optional)

1. Download this branch.
2. Run the project.
3. Go to any commons and verify.

## Tests

- [x] Backend Unit tests (`mvn test`) pass
- [x] Backend Test coverage (`mvn test jacoco:report`) 100%
- [x] Backend Mutation tests (`mvn test pitest:mutationCoverage`) 100% 
- [x] Frontend Unit tests (`npm test`) pass
- [x] Frontend Test coverage (`npm run coverage`) 100%
- [x] Frontend Mutation tests (`npx stryker run`) 100% 
- [x] Frontend Linting (`npx eslint --fix src`) 

## Dokku link
https://happycows-blobthebuilder.dokku-11.cs.ucsb.edu/

## Linked Issues

Closes #8

### CODE REVIEW

File: frontend/src/main/components/Commons/PagedProfitsTable.js [hunk 1]
```diff
@@ -84,19 +84,32 @@ const PagedProfitsTable = () => {
         []
     );
 
+    /*
+    style={{
+        width: column.width || "auto",
```
Comment (JohnLins) [inline frontend/src/main/components/Commons/PagedProfitsTable.js:89]: Please remove commented out code

**Review by JohnLins — COMMENTED** [review]
View comment

**Review by JohnLins — APPROVED** [review]
LGTM

**Review by pconrad — APPROVED** [review]
LGTM

**Pull request merged**